clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleServerGroup.groovy

```diff
@@ -113,6 +115,37 @@ class GoogleServerGroup {
           null
     }
 
+    /**
+     * @return The load balancing policy containing the
+     * capacity metrics and named ports this server group is configured with for all L7 backends.
+     */
+    GoogleHttpLoadBalancingPolicy getLoadBalancingPolicy() {
+      def loadBalancingPolicy = null
+      def loadBalancers = GoogleServerGroup.this.loadBalancers
+      loadBalancers.each { GoogleLoadBalancerView view ->
+        if (GoogleLoadBalancerType.valueOf(view.loadBalancerType) == GoogleLoadBalancerType.HTTP) {
+          GoogleHttpLoadBalancer.View httpView = view as GoogleHttpLoadBalancer.View
+          def backendServices = Utils.getBackendServicesFromHttpLoadBalancerView(httpView)
+          backendServices.each { GoogleBackendService backendService ->
+            def foundBackends = backendService.backends.findAll { GoogleLoadBalancedBackend backend ->
+              GCEUtil.getLocalName(backend.serverGroupUrl) == GoogleServerGroup.this.name
+            }
+            // Note: we configure the same load balancing policy for all backends on a create/clone,
```

Comment:
I guess the method name and comment should reflect that this is just one of many applicable load balancing policies, since that's not clear without this above comment.